CSS/style.css

```diff
@@ -133,11 +166,19 @@ footer {
   background-image: 'file:///Users/jkoehnke/Downloads/challenge/list-bg.png';
 }
 
+.author-photo {
+  float: left;
```

Comment:
You may want to try making this display: inline-block instead, then wrap the text in an inline-block div like I suggested for the header.